contrib/automation_tests/test_cases/symbol_locations.py

```diff
@@ -92,3 +92,19 @@ def _execute(self):
         self.expect_eq(0, len(symbol_path_list.descendants(control_type='ListItem')),
                        'List is empty')
         self.find_control('Button', 'Done', parent=ui).click_input()
+
+class EnableStadiaSymbolStore(E2ETestCase):
```

Comment:
I would maybe call this `ToggleStadiaSymbolStore` or maybe `ToggleEnableStadiaSymbolStore` and not give it a default of `False`. If you want to give it a default value, I think it should be `True`. Currently `EnableStadiaSymbolStore()` (without arguments) disables the symbol store and that feels wrong